content/guides/docker-concepts/building-images/understanding-image-layers.md

```diff
@@ -0,0 +1,210 @@
+
+With this layering, you can easily extend the images of others! As a developer, it allows you to focus on your unique application needs.
+
+### Merging the layers
+
+The magic for merging the layers lies in the use of a union filesystem. While this will get technical, here’s how it works:
+
+1. After each layer is downloaded, it is extracted into its own directory on the host filesystem. 
+2. A union filesystem is created. This stacks each of the layers' directories on top of each other into a new directory, giving a new and unified view. 
+3. When the container is started, its root directory is set to the location of this merged directory (using `chroot`).
+
+When the union filesystem is created, in addition to the image layers, a directory is created specifically for the running container. This allows the container to make filesystem changes while allowing the original image layers to remain untouched. This enables you to run multiple containers from the same underlying image.
+
+## Try it out
+
+In this hands-on, you will create new image layers manually using the [docker commit](https://docs.docker.com/reference/cli/docker/container/commit/) command. Note that you’ll rarely create images this way, as you’ll normally [use a Dockerfile](./writing-a-dockerfile.md). But, it makes it easier to understand how it’s all working.
+
+### Create a base image
+
+In this first step, you will create your own base image that you will then use for the following steps.
+
+1. [Download and install](https://www.docker.com/products/docker-desktop/) Docker Desktop.
+
+
+2. In a terminal, run the following command to start a new container:
+
+    ```console
+    $ docker run --name=base-container -ti ubuntu
+    ```
+
+    Once the image has been downloaded and the container has started, you should see a new shell prompt. This is running inside your container. It will look similar to the following (the container ID will vary):
+
+    ```console
+    root@d8c5ca119fcd:/#
+    ```
+
+3. Inside the container, run the following command to install Node.js:
+
+    ```console
+    $ apt update && apt install -y nodejs
+    ```
+
+    When this command runs, it downloads and installs Node inside the container. In the context of the union filesystem, these filesystem changes occur within the directory unique to this container. 
+
+4. Validate if Node is installed by running the following command:
+
+    ```console
+    $ node -e 'console.log("Hello world!")'
+    ```
+
+    You should then see a “Hello world!” appear in the console.
+
+5. Now that you have Node installed, you’re ready to save the changes you’ve made as a new image layer, from which you can start new containers or build new images. To do so, you will use the [docker commit](https://docs.docker.com/reference/cli/docker/container/commit/) command. Run the following command in a new terminal:
+
+    ```console
+    $ docker commit -m "Add node" base-container node-base
```

Comment:
I suggest we add the `container` subcommand here since that's the canonical command name and the `docker commit` shorthand is not one of the preferred shorthands.

```suggestion
5. Now that you have Node installed, you’re ready to save the changes you’ve made as a new image layer, from which you can start new containers or build new images. To do so, you will use the [`docker container commit`](https://docs.docker.com/reference/cli/docker/container/commit/) command. Run the following command in a new terminal:

    ```console
    $ docker container commit -m "Add node" base-container node-base
```